Documentation/configuration/metrics.rst

```diff
@@ -41,7 +41,17 @@ Endpoint
 Datapath
 --------
 
-* ``datapath_errors_total``: Total number of errors occurred in datapath management, labeled by area, name and address family.
+* ``datapath_errors_total``: Total number of errors occurred in datapath
+  management, labeled by area, name and address family.
+* ``datapath_conntrack_gc_runs_total``: Number of times that the conntrack
+  garbage collector process was run. It contains a label status that describes
+  if it was successful or not.
+* ``datapath_conntrack_gc_key_fallbacks_total``: Number of times that the Key fallback
+  was invalid.
+* ``datapath_conntrack_gc_entries``: The number of alive and deleted conntrack
+  entries at the end of a garbage collector run labeled by datapath family.
+* ``datapath_conntrack_gc_duration_seconds``: Duration in seconds of the garbage
+  collector process labelled by datapath and completation status.
```

Comment:
This is my fault 😭 labelled should be labeled to match US spelling, and the other descriptions we have. Sorry.

Reply:
done